yarn.lock

```diff
@@ -4663,6 +4698,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@mrleebo/prisma-ast@npm:^0.5.2":
+  version: 0.5.2
+  resolution: "@mrleebo/prisma-ast@npm:0.5.2"
+  dependencies:
+    chevrotain: ^10.4.2
+  checksum: 155795a245d885d6cd3edac43a3eb57c8ba5c178d71b7595e278c3f7879f78511b9796d3b13e37c228cfdba9621715a2af450611b68aa4d58739fbe129e8200d
+  languageName: node
+  linkType: hard
```

Comment:
You use the dmmf to grab this info, but it doesn't include the comments! So, I can't re-use that approach.